smdebug/tensorflow/reductions.py

```diff
@@ -1,5 +1,5 @@
 # Third Party
-import tensorflow as tf
+import tensorflow.compat.v1 as tf
```

Comment:
So are we dropping support for TF 1.13? In a different file this is wrapped in a try-except, but here it is not. Let's be consistent.

Reply:
the decision on TF 1.13 is still under discussion. there will be some modifications around this once the decision is made